docs/create-packages/Overview-and-Workflow.md

```diff
@@ -16,9 +16,9 @@ A package can also serve to only pull in any number of other dependencies, witho
 > [!Note]
 > When you create a package for use by other developers, it's important to understand that they are taking a dependency on your work. As such, creating and publishing a package also implies a commitment to fixing bugs and making other updates, or at the very least making the package available as open source so others can help to maintain it.
 
-Whatever the case, creating a package begins with deciding which assemblies and other files to package. You then create a manifest file, referred to as a `.nuspec` file, to describe the package's contents along with its identifier, version number, copyright information, MSBuild props and targets, and much more.
+Whatever the case, creating a package begins with deciding what it should contain along with its identifier, version number, copyright information, etc...
```

Comment:
```suggestion
Whatever the case, creating a package begins with deciding what it should contain, such as dependent files and assemblies, along with its identifier, version number, copyright information, and other information.
```

Reply:
Added a few suggestions; maybe you want to avoid saying anything about files and assemblies, since it might be specific to .nuspec? However, it would be nice to make "...what it should contain..." a bit less ambiguous, if possible. Also, generally we try to avoid "etc."

Reply:
Yes, I avoided assemblies as `dotnet pack` is project oriented (it works on a `.csproj` before the `.dll` has been created)

Reply:
Better to avoid mentioning "dependent files", too?

Reply:
Okay, I've tweaked the content a little bit. Please let me know what you think @Mikejo5000